PlayerControls/sources/ContentControlsViewController.swift

```diff
@@ -121,6 +121,14 @@ extension ContentControlsViewController.Props.Player.Item {
             public init() { }
         }
         
+        public var pictureInPicture: PictureInPicture = .unsupported
+        public enum PictureInPicture {
+            case unsupported
+            case supported(toggle: Action<Void>, state: PipState)
```

Comment:
We need to have toggle action in inactive case 

Maybe such struct be good?

`      public var pictureInPicture: PictureInPicture = PictureInPicture()
        public struct PictureInPicture {
            public var toggleAction: Action<Void>?
            public var state: State = .unsupported
            
            public enum State {
                case active, inactive, unsupported
            }
            public init() { }
        }`